wondrous-app/components/ProjectProfile/ListBounty.tsx

```diff
@@ -0,0 +1,34 @@
+import Grid from '@mui/material/Grid';
+import Compensation from 'components/Common/Compensation';
+import StarIcon from 'components/Icons/Sidebar/star.svg';
+import palette from 'theme/palette';
+import { ENTITIES_TYPES } from 'utils/constants';
+
+import { useEntityCreateButtonProps, useGetOrgEntity } from './helpers';
+
+const LeftComponent = ({ title }) => (
+  <Grid container gap="12px" alignItems="center" fontWeight="600" color={palette.white}>
+    {title}
+  </Grid>
+);
+
+const RightComponent = ({ rewards }) => <Compensation rewards={rewards} />;
+
+const useListBountyProps = () => ({
```

Comment:
I think these hooks are supposed to be more like component configs rather than hooks? custom hooks are designed to share state logic between components, in here the function is called only once, imo we can rewrite the hooks as normal functions and move them inside a helpers file

Reply:
I'll just use the ListWrapper component directly since this props will be passed to it anyway